---
layout: resources
title: Disease Resource Information
---

<h2><a name="Disease_Info" id="Disease_Info"></a>Defining Disease-Gene Associations</h2>

<h3>About GUDMAP Disease</h3>
<p>The GUDMAP Disease Database uses disease terms obtained from OMIM. In total <strong>8987</strong> OMIM Disease terms are contained within the database. </p>

<h3>Method - determining disease/gene associations</h3>

<p>Disease terms are obtained by querying OMIM for a range of genitourinary related terms (for example - <strong><em>genitourinary</em></strong>, <strong><em>GU</em></strong>, <strong><em>renal, nephron, urologic</em> etc.</strong>" within the <strong><em>Clinical Synopsis, Title and Text</em></strong> fields of the OMIM entry. The search is restricted to entries with either the <strong>#(pound)</strong>, <strong>%(percent)</strong> or <strong>'none' prefix</strong>. These three properties ensure that only entries that describe phenotypes are returned (avoiding gene entries). </p>
<p>Disease associations are determined for all genes that have in situ data in GUDMAP. In order to do this an additional text search for the orthologous human gene symbol within the main text of the OMIM entry is done. If the gene symbol is present an association between the disease and the gene is assumed. Associations are then assessed manually after including disease-gene associations from NCBI.</p>
<h4>NCBI evidence for gene-disease associations</h4>
<p>In addition to text-mining OMIM entries for evidence of disease-gene associations, associations are also obtained direct from NCBI via the <em>mim2genes</em> file - downloaded from the NCBI ftp site (<a href="https://ftp.ncbi.nlm.nih.gov/gene/DATA/">ftp://ftp.ncbi.nlm.nih.gov/gene/DATA/</a>). The <em>mim2genes</em> file is curated by NCBI and is a list of Entrez Gene IDs and OMIM IDs that are associated with one another.</p>
<!-- <p><strong>Assessing Disease-Gene Associations (Flags)</strong><br />
If an association is included in NCBIs <em>mim2genes</em> file then the association is considered valid and has a normal (N) flag. If the association is not included in the NCBI <em>mim2genes</em> file then the main text of the OMIM entry is check manually to assess whether the association is genuine or not. </p>
<ul>
  <li>If the evidence in the OMIM entry indicates that an association between the gene and the disease is likely, then the association is given a normal (N) flag. </li>
  <li>If there is limited or unclear evidence either way then the association is given an unsure (U) flag. </li>
  <li>In some cases a gene will be mentioned in a OMIM entry as having an association with a related disease, in this case a related (R) flag is given. An example of a related association could be CD2AP with Focal Segmental Glomerulosclerosis 1 (FSGS1) (OMIM 603278). Although CD2AP is mentioned in the entry for FSGS1, mutations in the gene are actually associated with/cause FSGS3 (OMIM 607832).</li>
</ul>-->
<h3>Notes</h3>
<p>  As a consequence of using a text-mining approach some gene symbols, such as 'T' and 'Rest' produces a large number of associations with OMIM entries, the majority of
  which are likely to be spurious. At present the associations for these types of genes are being checked manually and are not available via the search tools.</p>
<p>Also note that some associations between genes and diseases may not necessarily be positive - e.g. a gene and a disease could be associated because it is known that the gene <em>does not</em> play any role in disease (e.g. Adams-Oliver Syndrome and Msx1). Although the manual checking step should identify these negative associations, it may not always be the case. It is important therefore to follow the link to OMIM to view a more in depth description of the association.</p>

<h2><a name="Phenotype_Info" id="Phenotype_Info"></a>Phenotype-Gene Associations</h2>
<h3>About phenotype associations</h3>
<p>The phenotype-gene associations are obtained from the <a href="http://www.informatics.jax.org/phenotypes.shtml" target="_blank">MGI (Mouse Genome Informatics)</a>. A gene is considered associated with a particular Mammalian Phenotype (MP) term if a mutant allele of that gene has been annotated with that MP term.</p>

<h3>Derived annotations</h3>
<p>When search for genes associated with a phenotpye term: If a gene has a derived annotation to the MP search term it means that the gene is directly annotated to a different MP term that, according to the MGI MP Ontology, is a child term of the MP search term.</p>

<p>For example, the gene <strong>Acvr2a</strong> has a derived annotation to <strong>abnormal corpus luteum</strong> due to the fact that <strong>Acvr2a</strong> is directly annotated to <strong>absent corpus luteum</strong> â€“ which, according to the MGI MP Ontology is a child term of <strong>abnormal corpus luteum</strong>. </p>

<p>When searching for phenotpye terms associated with a gene: Where an MP term has a derived annotation to the gene it is because the gene is directly annotated with a different MP term which, according to the MGI MP Ontology is a child of the other MP term. </p>

<p>For example, <strong>abnormal gametogenesis</strong> has derived annotation to <strong>Wnt4</strong> because <strong>Wnt4</strong> is directly annotated with <strong>abnormal oogenesis</strong>, which, according to the MGI MP Ontology is child term of <strong>abnormal gametogenesis</strong>.</p>
