---
title: "New datasets from Humphreys lab and new publication from Cantley lab"
layout: post
tag: rbk
---

Here are some new releases from the RBK consortium. We hope you're enjoying the start of your summer!

**Two new datasets from Ben Humphrey's lab**

[16-E1GE - Transcriptional profiling of mouse proximal tubular epithelial cells during injury and repair](https://www.rebuildingakidney.org/id/16-E1GE)

Dr. Ben Humphrey's lab used Translating Ribosome Affinity Purification coupled with next generation sequencing (TRAP-seq) to profile mouse tubular epithelial cells during the course of injury and repair using the Kim-GCE mouse model. This novel mouse model allows lineage tracing of injured mouse tubular epithelial cells, which upregulate Kim1, and translational profiling by ribosomal pull down when crossed to the EGFP-L10a mouse. Transcriptional profiling was done at day 2, 7, and 14 after bilateral ischemia reperfusion injury.

[16-E08W - Translational profiles of proximal tubule during UUO](https://www.rebuildingakidney.org/id/16-E08W)
    
Injury to the proximal tubule can initiate chronic kidney disease and rates of progression are approximately 50% faster in males compared to females. The precise transcriptional changes in this nephron segment during fibrosis, and potential differences between sexes, remain undefined. Humphrey's lab generated mice with proximal tubule-specific expression of an L10a ribosomal subunit protein fused with eGFP. They performed unilateral ureteral obstruction (UUO) surgery on male (n = 4) and female (n = 3) mice, collected proximal tubule-specific and bulk cortex mRNA at day 5 or 10 and sequenced to a depth of 30 million reads. Computational methods were applied to identify both sex-biased and shared molecular responses to fibrotic injury, including up- and down-regulated long non-coding RNAs and transcriptional regulators. Critical genes and pathways were validated by in situ hybridization.

**New publication from Cantley lab.**

Lloyd Cantley's lab has published a new paper, [Development of a 2-dimensional atlas of the human kidney with imaging mass cytometry](https://insight.jci.org/articles/view/129477), on _JCI Insight_. You can [read the whole paper here](https://insight.jci.org/articles/view/129477).

The following is the abstract:

>An incomplete understanding of the biology of the human kidney, including the relative abundances of and interactions between intrinsic and immune cells, has long constrained the development of therapies for kidney disease. The small amount of tissue obtained by renal biopsy has previously limited the ability to use patient samples for discovery purposes. Imaging mass cytometry (IMC) is an ideal technology for quantitative interrogation of scarce samples, permitting concurrent analysis of more than 40 markers on a single tissue section. Using a validated panel of metal-conjugated antibodies designed to confer unique signatures on the structural and infiltrating cells comprising the human kidney, we performed simultaneous multiplexed imaging with IMC in 23 channels on 16 histopathologically normal human samples. We devised a machine-learning pipeline (Kidney-MAPPS) to perform single-cell segmentation, phenotyping, and quantification, thus creating a spatially preserved quantitative atlas of the normal human kidney. These data define selected baseline renal cell types, respective numbers, organization, and variability. We demonstrate the utility of IMC coupled to Kidney-MAPPS to qualitatively and quantitatively distinguish individual cell types and reveal expected as well as potentially novel abnormalities in diseased versus normal tissue. Our studies define a critical baseline data set for future quantitative analysis of human kidney disease

