---
title: "New publications from McMahon, Bonventre and Humphreys labs"
layout: post
tag: rbk
---

Here are three new publications from members of the RBK consortium:

### Spatial transcriptomics defines injury-specific microenvironments in the adult mouse kidney and novel cellular interactions in regeneration and disease [preprint]

*Authors:* Polonsky, Michal; Gerhardt, Louisa M. S.; Yun, Jina; Koppitch, Kari; Colón, Katsuya Lex; Amrhein, Henry; Zheng, Shiwei; Yuan, Guo-Cheng; Thomson, Matt; Cai, Long and McMahon, Andrew P.

*Journal:* Molecular Biology

*URL:* [https://www.doi.org/10.1101/2023.11.22.568217](https://www.doi.org/10.1101/2023.11.22.568217)

*Summary:* Kidney injury disrupts the intricate renal architecture and triggers limited regeneration, and injury-invoked inflammation and fibrosis. Deciphering molecular pathways and cellular interactions driving these processes is challenging due to the complex renal architecture. Here, we applied single cell spatial transcriptomics to examine ischemia-reperfusion injury in the mouse kidney. Spatial transcriptomics revealed injury-specific and spatially-dependent gene expression patterns in distinct cellular microenvironments within the kidney and predicted Clcf1-Crfl1 in a molecular interplay between persistently injured proximal tubule cells and neighboring fibroblasts. Immune cell types play a critical role in organ repair. Spatial analysis revealed cellular microenvironments resembling early tertiary lymphoid structures and identified associated molecular pathways. Collectively, this study supports a focus on molecular interactions in cellular microenvironments to enhance understanding of injury, repair and disease.

One-Sentence Summary:
Spatial transcriptomics predicted a molecular interplay amongst neighboring cell-types in the injured mammalian kidney

### 3D Hydrogel Encapsulation Regulates Nephrogenesis in Kidney Organoids

*Authors:* Nerger, Bryan A.; Sinha, Sumit; Lee, Nathan N.; Cheriyan, Maria; Bertsch, Pascal; Johnson, Christopher P.; Mahadevan, L.; Bonventre, Joseph V. and Mooney, David J.

*Journal:* Advanced Materials. 2308325. January 2024.

*URL:* [https://doi.org/10.1002/adma.202308325](https://doi.org/10.1002/adma.202308325)

*Summary:* Stem cell‐derived kidney organoids contain nephron segments that recapitulate morphological and functional aspects of the human kidney. However, directed differentiation protocols for kidney organoids are largely conducted using biochemical signals to control differentiation. Here, the hypothesis that mechanical signals regulate nephrogenesis is investigated in 3D culture by encapsulating kidney organoids within viscoelastic alginate hydrogels with varying rates of stress relaxation. Tubular nephron segments are significantly more convoluted in kidney organoids differentiated in encapsulating hydrogels when compared with those in suspension culture. Hydrogel viscoelasticity regulates the spatial distribution of nephron segments within the differentiating kidney organoids. Consistent with these observations, a particle‐based computational model predicts that the extent of deformation of the hydrogel–organoid interface regulates the morphology of nephron segments. Elevated extracellular calcium levels in the culture medium, which can be impacted by the hydrogels, decrease the glomerulus‐to‐tubule ratio of nephron segments. These findings reveal that hydrogel encapsulation regulates nephron patterning and morphology and suggest that the mechanical microenvironment is an important design variable for kidney regenerative medicine.

### Epigenetic reprogramming driving successful and failed repair in acute kidney injury [preprint]

*Authors:* Yoshiharu Muto, Eryn E Dixon, Yasuhiro Yoshimura, Nicolas Ledru, Yuhei Kirita,  Haojia Wu and Benjamin D Humphreys

*Journal:* BioRxiv

*URL:* [https://www.biorxiv.org/content/10.1101/2024.01.20.576421v2](https://www.biorxiv.org/content/10.1101/2024.01.20.576421v2)

*Summary:* Acute kidney injury (AKI) causes epithelial damage followed by subsequent repair. While successful repair restores kidney function, this process is often incomplete and can lead to chronic kidney disease (CKD) in a process called failed repair. To better understand the epigenetic reprogramming driving this AKI-to-CKD transition we generated a single nucleus multiomic atlas for the full mouse AKI time course, consisting of ~280,000 single nucleus transcriptomes and epigenomes. We reveal cell-specific dynamic alterations in gene regulatory landscapes reflecting especially activation of proinflammatory pathways. We further generated single nucleus multiomic data from four human AKI samples including validation by genome-wide identification of NF-kB binding sites. A regularized regression analysis identifies key regulators involved in both successful and failed repair cell fate, identifying the transcription factor CREB5 as a regulator of both successful and failed tubular repair that also drives proximal tubule cell proliferation after injury. Our interspecies multiomic approach provides a foundation to comprehensively understand cell states in AKI.


[*Browse through more RBK-related publications here.*](https://www.atlas-d2k.org/chaise/recordset/#2/Common:Publication/*::facets::N4IghgdgJiBcDaoDOB7ArgJwMYFM6JHQBcAjdafEAYRQFtaUIQAaEABTRIBsBLLMIj0YB9GhFQZBaWsIBmAaxwBPEAF0AvqwBKASQAia1lgAWKPjiSUAigFoAzAGkArABY16jUA@sort(Year::desc::,Month::desc::,RID::desc::))
